src/studio/editors/code.c

```diff
@@ -2959,7 +2959,7 @@ static void processKeyboard(Code* code)
     bool usedKeybinding = true;
 
     // handle bookmarks
-    if(keyWasPressed(code->studio, tic_key_f1))
+    if(keyWasPressed(code->studio, tic_key_f7))
```

Comment:
We already use the `F7` hotkey to save the cover image for the game but only in game mode. 
To fix the #1986 issue I think we need to filter `F1` presses when you switch to the code editor using hotkeys,
I mean don't allow bookmarking when you just switched from other editor using `F1`.

Reply:
It would also be a good solution. Useful too to fix the android problem when pressing ctrl + s writes s. Don't know why that one doesn't happen on desktop.

Reply:
Just made a proper fix for this, I'll submit it